SnapWar/Entities/SWUserEntity.swift

```diff
@@ -0,0 +1,12 @@
+class SWUserEntity {
+    
+    static let shared = SWUserEntity()
+    
+    var externalId: String = ""
```

Comment:
in general, we should use Optional for anything that might be not initialized and use try/guard to check if they're set instead of comparing with empty string. this is the best practice!